test/smart-account/SA.Modules.specs.ts

```diff
@@ -81,17 +81,99 @@ describe("NEW::: Ownerless Smart Account Modules: ", async () => {
     });
 
     // can not enable address(0) as module
+    it("Can not enable address(0) as module", async()=> {
+      const {
+        ecdsaModule,
+        userSA,
+        entryPoint
+      } = await setupTests();
+
+      let userOp = await makeecdsaModuleUserOp(
+        "enableModule",
+        [AddressZero],
+        userSA.address,
+        smartAccountOwner,
+        entryPoint,
+        ecdsaModule.address
+      );
+
+      const tx = await entryPoint.handleOps([userOp],alice.address);
+      expect(tx).to.emit(entryPoint,"UserOperationRevertReason");
+
+    });
 
     // can not enable sentinel module
+    const sentinel_address = "0x0000000000000000000000000000000000000001";
```

Comment:
1) use camelCase for names
2) declaration should not be located outside of the test statement. The best option is to put it on line 20; thus, it will be available for the second test (Can not setup and enable sentinel module) as well.